dist/net.unvanquished.Unvanquished.appstream.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021 Antoine Fontaine <afontain@gnugen.ch> -->
+<component type="desktop-application">
+  <id>net.unvanquished.Unvanquished</id>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <project_license>MIT AND GPL-3.0-only AND Zlib AND BSD-3-Clause AND CC0-1.0 AND CC-BY-2.5 AND CC-BY-SA-2.5 AND CC-BY-SA-3.0 AND CC-BY-SA-4.0</project_license>
+  <name>Unvanquished</name>
+  <summary>A fast paced, first person strategy game</summary>
+
+  <description>
+    <p>
+      Unvanquished is a strategy game played on the first person.
+      Become a marine, handle your big weapon and dress that
+      impressive exoskeleton to save the human race like a boss. Or if
+      you prefer, be a monstrous beast and save those cute grangers
+      from those two-legged invaders who only know how to turn
+      beautiful organic equilibrium into lifeless machines and dead
+      concreted worlds.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">net.unvanquished.Unvanquished.desktop</launchable>
```

Comment:
Isn't upstream [desktop file](https://github.com/Unvanquished/Unvanquished/search?q=.desktop) name just unvanquished.desktop? Also, this desktop file is not present in the _dist_ directory, only in distro specific directories.

Reply:
I believe the Debian package is dead, but @Viech still updates the Arch one.